tests/test_visualizations.py

```diff
@@ -0,0 +1,308 @@
+              "created": "2017-01-17T00:20:08.805598+00:00",
+              "currency": "USD",
+              "data": {
+                "ref_id": "c28046b7-5a5d-43a9-9324-7327bbd934de"
+              },
+              "data_type": "csv",
+              "description": "Nova Scotia Canada EQ COM",
+              "event_catalogs": [
+                {
+                   "ref_id": "c508b867-6d6b-4eaf-aad5-294c4debf6de"
+                }
+              ],
+              "id": "8bfc42b8-75ed-450a-b474-4f936b449dc3",
+              "loss_type": "LossGross",
+              "modified": "2017-01-17T00:20:18.231491+00:00",
+              "profile": {
+                "attributes": {
+                  "Country": ["Canada"],
+                  "Peril": ["EQ"],
+                  "Region": ["North America"],
+                  "SubRegion": ["NB", "NS","PE"]
+                },
+                "avg_annual_loss": 5591.573225,
+                "currency": "USD",
+                "max_loss": 51649446.69,
+                "min_loss": 4.66,
+                "non_zero_losses": 24,
+                "num_losses": 24
+              },
+              "start_date": "2017-01-01T00:00:00+00:00",
+              "status": "processing_succeeded",
+              "trial_count": 10000
+            }
+          ],
+          "meta_data": {},
+          "nth": 1,
+          "participation": 1.0,
+          "reinstatements": [
+            {
+              "brokerage": 0.1,
+              "premium": 1.0
+              }
+          ]
+        },
+        "target_currency": "USD",
+        "ylt_id": "9d61ecf9-9219-57ba-dd2a-fa515732639e"
+    }
+    return convert_to_analyzere_object(content, LayerView)
+
+@pytest.fixture(scope='session')
+def layer_view_attachment_warning():
+    """Unlimited attachments are warnings"""
+    content = {
+        "id": str(uuid.uuid4()),
+        "_type": "CatXL",
+        "attachment": {
+            "currency": "USD",
+            "value": float_info.max,
+        }
+    }
+    return convert_to_analyzere_object(content, LayerView)
+
+@pytest.fixture(scope='session')
+def layer_view_participation_warning():
+    """No participation is a warning"""
+    content = {
+        "_type": "CatXL",
+        "participation": 0.0
+    }
+    return convert_to_analyzere_object(content, LayerView)
+
+@pytest.fixture(scope='session')
+def layer_view_filter_warning():
+    """A FilterLayer with no filters and invert = false is a warning"""
+    content = {
+        "_type": "FilterLayer",
+        "filters": [],
+        "invert": False
+    }
+    return convert_to_analyzere_object(content, LayerView)
+
+
+class TestDescriptionFormatter:
+    def test_description_with_colon(self):
+        raw = 'auto: FilterLayer USHU'
+        expected = 'auto: FilterLayer USHU'
+        assert visualizations._format_description(raw) == expected
+
+    def test_description_with_windows_file_path(self):
+        raw = 'Layer 2 : Z:\\losses\\alpha.csv'
```

Comment:
>>> r'Z:\\losses\\alpha.csv'
'Z:\\\\losses\\\\alpha.csv'

Using r'' for your string delimiters can make things easier on the eyes with escapes (r mnemonic == raw string; i.e. no escape interpolation)